mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModel.kt

```diff
@@ -399,7 +416,8 @@ abstract class MavericksViewModel<S : MavericksState>(
         } else {
             flowWhenStarted(lifecycleOwner)
         }
-        val scope = lifecycleOwner?.lifecycleScope ?: viewModelScope
+
+        val scope = lifecycleOwner?.lifecycleScope?.let { it + configFactory.subscriptionCoroutineContextOverride } ?: viewModelScope
```

Comment:
nit, probably it looks slightly better 
```suggestion
        val scope = lifecycleOwner?.lifecycleScope?.plus(configFactory.subscriptionCoroutineContextOverride) ?: viewModelScope
```

btw don't you need to add it to case when viewModelScope is used? (MavericksView is only an example in the subscriptionCoroutineContextOverride doc: 
>Provide a context that will be added to the coroutine scope when a subscription is registered

)

Reply:
nice syntax idea. 

good catch, I suppose we should add it in both cases. Previously it didn't matter since it was the same context, but that changed once I added a dedicated `subscriptionCoroutineContextOverride`